### PR TITLE
Danilg/sound sense

### PR DESCRIPTION
**Pull request by DarkMinimum**
- remove damage config
- add hear config
- add handling Moise of projectile and items
- add Noise Trigger

### CODE REVIEW

File: Source/CryptRaider/Actor/Destructible/Projectile.cpp [hunk 1]
```diff
@@ -37,6 +37,14 @@ void AProjectile::Charge()
 	bIsCharged = true;
 }
 
+void AProjectile::Tick(float DeltaTime)
+{
+	if (!Tags.Contains(GrabbedTag))
```
[the remark this answers is not in the document]
Comment (DarkMinimum) [reply Source/CryptRaider/Actor/Destructible/Projectile.cpp]: In case of using OnHit, we can remove this block

**Comment by Iluhap** [comment]
Lets parametrize all the hardcoded `Noise` volume values and `Velocity` lenght

**Review by Iluhap — APPROVED** [review]
Lets fix minor comments then merge

**Review by Iluhap — DISMISSED** [review]
BasePlayerController was updated, merge conflict need to be resolved. Then Merge

**Review by Iluhap — APPROVED** [review]
Nice

**Pull request merged**